code-of-conduct.md

```diff
@@ -1,3 +1,3 @@
 # Kubernetes Community Code of Conduct
 
-Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
+Kubernetes follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
I believe the idea was to make sure that the CoC at k/community acts as the authoritative source for all repos in the Kubernetes org. Ref https://github.com/kubernetes/community/issues/1527, https://github.com/kubernetes/kubernetes-template-project/pull/13.